doc/content/components.md

```diff
@@ -0,0 +1,46 @@
+# Key Components
+
+## Description
+
+Documentation on the main components comprising Platypus can be found in:
```

Comment:
Maybe mention that the Platypus components all need to be used together and shouldn't be mixed with MOOSE normal classes (with the exception of functions). If you want to use them with the MOOSE components then they need to be put in separate sub-problems.